docs/source/histoqc.md

```diff
@@ -0,0 +1,72 @@
+# Instructions to setup and run HistoQC on H&E slides datasets
+
+## Install Openslide
+
+```shell
+sudo apt-get install openslide-tools
+sudo apt-get install python3-openslide
+```
+
+## Clone HistoQC
+
+```shell
+git clone https://github.com/choosehappy/HistoQC
+```
+
+Then edit `HistoQC/environment.devenv.yml` and remove line 17
```

Comment:
can you please specify what line 17 contains? line 17 content may change in histo qc repo...

Reply:
Done now, thanks